### PR TITLE
Logger Injection pt. 6

### PR DESCRIPTION
**Pull request by jmank88**
Towards: https://app.shortcut.com/chainlinklabs/story/18500

### CODE REVIEW

File: core/logger/orm.go [hunk 1]
```diff
@@ -14,20 +14,21 @@ type ORM interface {
 }
 
 type orm struct {
-	db *sqlx.DB
+	db   *sqlx.DB
+	lggr Logger
```
Comment (samsondav) [inline core/logger/orm.go:18]: I still think its something of an antipattern to have a logger "ORM" at all.

**Comment by samsondav** [comment]
@cl-homu r+

**Comment by cl-homu** [comment]
:pushpin: Commit 49ec2e922a0282061c3237641e3cc4aeb75a8207 has been approved by `samsondav`

**Comment by cl-homu** [comment]
:hourglass: Testing commit 49ec2e922a0282061c3237641e3cc4aeb75a8207 with merge 1e09c87d9a3aaba331a2fda5b8a19dfae3dc9776...

**Comment by cl-homu** [comment]
:broken_heart: Test failed - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4211579438?check_suite_focus=true)

**Comment by samsondav** [comment]
@cl-homu retry

**Comment by cl-homu** [comment]
:hourglass: Testing commit 49ec2e922a0282061c3237641e3cc4aeb75a8207 with merge d5fb713d0cb71ca31c3c4c63a882797309707650...

**Comment by cl-homu** [comment]
:broken_heart: Test failed - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4211759466?check_suite_focus=true)

**Comment by samsondav** [comment]
@cl-homu retry

**Comment by cl-homu** [comment]
:hourglass: Testing commit 49ec2e922a0282061c3237641e3cc4aeb75a8207 with merge 5adcbccccdf2e305f9b5b7b79c8ad62a8006e53f...

**Comment by cl-homu** [comment]
:sunny: Test successful - [checks-core](https://github.com/smartcontractkit/chainlink/runs/4211963708?check_suite_focus=true), [checks-lint](https://github.com/smartcontractkit/chainlink/runs/4211963557?check_suite_focus=true), [checks-npm](https://github.com/smartcontractkit/chainlink/runs/4211963924?check_suite_focus=true), [checks-operator](https://github.com/smartcontractkit/chainlink/runs/4211963713?check_suite_focus=true), [checks-prettier](https://github.com/smartcontractkit/chainlink/runs/4211963898?check_suite_focus=true), [checks-solidity](https://github.com/smartcontractkit/chainlink/runs/4211963880?check_suite_focus=true)
Approved by: samsondav
Pushing 5adcbccccdf2e305f9b5b7b79c8ad62a8006e53f to develop...

**Pull request merged**